src/odemis/gui/cont/streams.py

```diff
@@ -331,6 +336,40 @@ def _add_stream_setting_controls(self):
             conf = stream_config.get(vaname)
             self.add_setting_entry(vaname, va, hw_comp=None, conf=conf)
 
+    def _add_axis_controls(self):
+        """
+        Add controls for the axes that are connected to the stream
+        """
+        
+        self._axes = self.stream.axis_map
+
+        if self._axes is not None:
```

Comment:
It shouldn't need to use `axis_map`, just use `axis_vas`, and add each VA as an entry. Done.